closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -219,16 +222,18 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_loadCrypto
     int ossl_ver;
 
     /* Load OpenSSL Crypto library */
-    crypto_library = load_crypto_library();
+    crypto_library = load_crypto_library(traceEnabled);
     if (NULL == crypto_library) {
-        /* fprintf(stderr, " :FAILED TO LOAD OPENSSL CRYPTO LIBRARY\n"); */
-        /* fflush(stderr); */
+        if (1 == traceEnabled) {
```

Comment:
`traceEnabled` should be of type `jboolean` and the test should be
```
    if (traceEnabled) {
```
or
```
    if (JNI_FALSE != traceEnabled) {
```

Reply:
Modified